.github/workflows/packaging.yml

```diff
@@ -114,7 +192,12 @@ jobs:
       - name: Build and test
         shell: bash
         run: |
-          docker-compose run java-dist
+          pushd adbc/
+          docker-compose run \
+            -e SETUPTOOLS_SCM_PRETEND_VERSION=${VERSION} \
```

Comment:
Similarly I don't think this affects Java at all

Reply:
You're right.